gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -166,6 +168,71 @@ def __ge__(self, other):
 
     return other <= self
 
+  def to_vcf_line(self):
```

Comment:
Consider moving this to _VcfSink. The reason is that once we add support to genomics protos, then it would be more natural to have the convert logic in one place (we can't add methods to the proto class).

Reply:
Good point, keeping the Variant interface as simple as possible makes sense.